jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/BufferedResponseHandler.java

```diff
@@ -56,12 +55,14 @@
 public class BufferedResponseHandler extends Handler.Wrapper
 {
     public static final String BUFFER_SIZE_ATTRIBUTE_NAME = BufferedResponseHandler.class.getName() + ".buffer-size";
+    public static final int DEFAULT_BUFFER_SIZE = 16384;
 
     private static final Logger LOG = LoggerFactory.getLogger(BufferedResponseHandler.class);
 
     private final IncludeExclude<String> _methods = new IncludeExclude<>();
     private final IncludeExclude<String> _paths = new IncludeExclude<>(PathSpecSet.class);
     private final IncludeExclude<String> _mimeTypes = new IncludeExclude<>();
+    private int bufferSize = DEFAULT_BUFFER_SIZE;
```

Comment:
If you look at the implementation of BufferedOutputStream you see that it has a min and max buffer size.   I think there could be value in that approach and small single writes will not end up in huge buffers.

Reply:
Good idea, I'll work on that.

Reply:
Done; I've taken 1 KB as the min buffer size for no particular reason, so if you think a smaller one would make sense, I'm open to suggestions.